lua/efmls-configs/formatters/yq.lua

```diff
@@ -0,0 +1,13 @@
+-- Metadata
+-- languages: yaml
+-- url: https://github.com/mikefarah/yq
+
+local fs = require('efmls-configs.fs')
+
+local formatter = 'yq'
+local args = '. ${INPUT}'
```

Comment:
Can you wrap the `${INPUT}` in quotes, as mentioned in this PR #72 assuming the filepath can have spaces.

Reply:
Done.